docs/Fields.md

```diff
@@ -126,12 +126,13 @@ import { BooleanField } from 'react-admin';
 
 #### Properties
 
-| Prop              | Required | Type             | Default                    | Description                       |
-| ----------------- | -------- | ---------------- | -------------------------- | --------------------------------- |
-| `valueLabelTrue`  | Optional | string           | 'true'                     | Aria label for the truthy value   |
-| `valueLabelFalse` | Optional | string           | 'false'                    | Aria label for the falsy value    |
-| `TrueIcon`        | Optional | SvgIconComponent | `@material-ui/icons/Done`  | Icon to show for the truthy value |
-| `FalseIcon`       | Optional | SvgIconComponent | `@material-ui/icons/Clear` | Icon to show for the falsy value  |
+| Prop              | Required | Type             | Default                    | Description                                                      |
+| ----------------- | -------- | ---------------- | -------------------------- | ---------------------------------------------------------------- |
+| `valueLabelTrue`  | Optional | string           | 'true'                     | Aria label for the truthy value                                  |
+| `valueLabelFalse` | Optional | string           | 'false'                    | Aria label for the falsy value                                   |
+| `TrueIcon`        | Optional | SvgIconComponent | `@material-ui/icons/Done`  | Icon to show for the truthy value                                |
+| `FalseIcon`       | Optional | SvgIconComponent | `@material-ui/icons/Clear` | Icon to show for the falsy value                                 |
+| `strictTypeCheck` | Optional | boolean          | `true`                     | If `true` the field's value is evaluated strictly as a `boolean` |
```

Comment:
I'd prefer if the prop worked the other way around: st it to true to *enable* loose values. Besides, the second part of the prop name is wrong, it's not a check and it's not about type, but value. I'd call the prop `looseValue`.